src/ui/server/controllers/insurance/feedback/feedback-form/index.ts

```diff
@@ -87,6 +101,8 @@ const post = async (req: Request, res: Response) => {
       });
     }
 
+    const referralUrl = req.flash('feedbackOriginUrlDB');
```

Comment:
Seems we don't need `feedbackOriginUrlDB` as it's got the same value as `req.flash('feedbackOriginUrl')`?